spec/fixtures/requests/koi8-r.txt

```diff
@@ -2,7 +2,7 @@ HTTP/1.1 200 OK
 Server: nginx/1.11.10
 Date: Tue, 04 Jul 2017 16:43:39 GMT
 Content-Type: text/html
-Content-Length: 273
+Content-Length: 384
```

Comment:
@akihikodaki It seems that this value was wrong, because when I checked, the HTML loaded was cut-off abruptly. This caused an error in the koi8-r test suite that was not related to the encoding, but to an unclosed tag. (I think Nokogiri swallowed the error)